City-of-Helsinki/helsinki-paatokset
# PP-333: Update TPR module to most recent, add contacts block to polic…

Update tpr module to latest  + add contacts block to policymaker pages.

To test: 

- Checkout branch. Run `make build; make drush-cim; make drush-cr`
- Run the tpr migration: `drush mim tpr_unit`. This takes a whole lot of time, but since we don't need the all of them you can use `--limit` flag or just let it run for a while and then kill it
- Once you have some units, edit a policymaker and link 2+ contacts via **Contact information** field
- Check the modified policymaker page. You should see the contacts listed in the sidebar region (left side of the page on desktop, below content region in mobile).
